README.md

```diff
@@ -1455,6 +1455,28 @@ Options for the [OpenId Connect](http://developer.okta.com/docs/api/resources/oi
 
 ## Can be in Client Configuration
 
+The following configuration options can be included **only** when instantiating Okta Auth JS (`new OktaAuth(config)`).
+
+- **tokenManager:** Specify the type of storage for tokens. Defaults to [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) and will fall back to [sessionStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage), and/or [cookie](https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie) if not the previous type is not available.
+
+  ```javascript
+  tokenManager: {
+    {
+      storage: 'sessionStorage'
+    }
+  }
+  ```
+
+  The `tokenManager` will **automatically refresh tokens** for you when they expire. To disable this feature, set `autoRefresh` to false.
+
+  ```javascript
+  tokenManager: {
+    {
+      autoRefresh: false
+    }
+  }
+  ```
+
```

Comment:
It's good that we're adding this section.  Looking at both these sections it's hard to understand what's being explained, because they are just a bunch of orphaned key/value pairs without hints of where they go.  Showing them with the constructor call or a relevant method call would would help.